client/trino-jdbc/src/main/java/io/trino/jdbc/ConnectionProperties.java

```diff
@@ -440,6 +442,15 @@ public AccessToken()
         }
     }
 
+    private static class TimeZoneID
+            extends AbstractConnectionProperty<String>
+    {
+        public TimeZoneID()
+        {
+            super("TimeZoneID", NOT_REQUIRED, ALLOWED, STRING_CONVERTER);
```

Comment:
"Timezone" to be consistent with CLI's ``--timezone``

Reply:
Let's make it lowercase `timezone` to be consistent with other properties (all are lowercase except Kerberos and SSL)